iOS-SDKTests/iOS_SDKTests.swift

```diff
@@ -35,13 +35,21 @@ class iOS_SDKTests: XCTestCase {
     }
     
     func testDateLocalization() {
-        //Set the expected result in format # days, using the app settings localization
-        let expectedResult: String = "5 dagen"
+        //Check the device language
+        let languageSetting = SettingsManager.get(type: .language)
+        guard languageSetting.code == "en" else {
+            XCTFail("The phone language isn't in english")
+            return
+        }
+        
+        //Set the expected result in format # days, for the test the language setting is expected in english
+        let expectedResult: String = "5 days"
+        
         //Set in seconds the time in the past for the date to compare. Example 3,600 == 1 hour, 360,000 == 100 hours
-        let secondsInThePast: Double = 432000 //This number should be seconds and in positive
+        let secondsInThePast: Double = 432001 //This number should be seconds and in positive
         
         //Don't edit beyond this point
-        let timeInterval = TimeInterval(-secondsInThePast)
+        let timeInterval = TimeInterval(secondsInThePast)
```

Comment:
Why did you remove the `-`?

Reply:
The code in the app to fix the label was fixed, but when I did it, I didn't updated the test, this change takes in consideration the app fix.

As you can see in this commit https://github.com/schedjoules/ios-public-calendars-sdk/commit/f6a6fb797d4ba1f6c897beed58c394e11c7db877#diff-f8c16d5f3afbe4619171becfdf801aab in DateExtensions first we were comparing the calendar date (self) to the current date Date(), but in that commit the formatter was changed to `Date() to self`